lib/process-services-cloud/src/lib/form/services/form-cloud.service.ts

```diff
@@ -171,9 +171,21 @@ export class FormCloudService extends BaseCloudService implements FormCloudServi
      */
     getTaskVariables(appName: string, taskId: string): Observable<TaskVariableCloud[]> {
         const apiUrl = `${this.getBasePath(appName)}/query/v1/tasks/${taskId}/variables`;
-
-        return this.get(apiUrl).pipe(
-            map((res: any) => res.list.entries.map((variable) => new TaskVariableCloud(variable.entry)))
+        let skipCount = 0;
+        let maxItems = 1000;
+
+        return this.get(apiUrl, { maxItems: maxItems, skipCount: skipCount }).pipe(
```

Comment:
we can use Object Property Shorthand for syntactic sugar
```suggestion
        return this.get(apiUrl, { maxItems, skipCount }).pipe(
```